src/host/main.cpp

```diff
@@ -117,10 +117,19 @@ int main(int argc, char** argv)
     "Address to advertise publicly to clients (defaults to same as "
     "--rpc-address)");
 
-  std::string ledger_file("ccf.ledger");
-  app.add_option("--ledger-file", ledger_file, "Ledger file")
+  std::string ledger_dir("ledger");
+  app.add_option("--ledger-dir", ledger_dir, "Ledger directory")
     ->capture_default_str();
 
+  size_t ledger_chunk_threshold = 5'000'000;
+  app
+    .add_option(
+      "--ledger-chunk-max-bytes",
+      ledger_chunk_threshold,
+      "Minimum size (bytes) at which a new ledger chunk is created.")
+    ->capture_default_str()
+    ->transform(CLI::AsSizeValue(true)); // 1000 is kb
```

Comment:
The comment is slightly mysterious, do we need it?

Reply:
The comment refers to the `true` which indicates that `k` is 1000 and not 1024.

Reply:
Ah, got it, sorry!

Reply:
I was very confused by it too at first and had to look into `CLI11.hpp` directory to find this out.